vm/devices/vmbus/vmbus_server/src/lib.rs

```diff
@@ -870,12 +877,17 @@ impl ServerTask {
                         .merge(&self.server.with_notifier(&mut self.inner));
                 });
             }
-            VmbusRequest::Save(rpc) => rpc.handle_sync(|()| SavedState {
-                server: self.server.save(),
-            }),
-            VmbusRequest::Restore(rpc) => {
-                rpc.handle_sync(|state| self.server.restore(state.server))
+            VmbusRequest::Save(rpc) => {
+                let lost_synic_bug_fixed = true;
+                rpc.handle_sync(|()| SavedState {
+                    server: self.server.save(),
+                    lost_synic_bug_fixed,
```

Comment:
Just use `lost_synix_bug_fixed: true,` inline, there's no need to introduce a local.